packages/app-studio-toolkit/src/utils/bas-utils.ts

```diff
@@ -58,3 +64,34 @@ function getExtensionRunPlatform(): ExtensionRunMode {
 
   return runPlatform;
 }
+
+export async function reportProjectTypesToUsageAnalytics(
+  basToolkitAPI: BasToolkit
+) {
+  const devspaceInfo = await devspace.getDevspaceInfo();
+  const projects = await getProjectsInfo(basToolkitAPI);
+  void AnalyticsWrapper.traceProjectTypesStatus(
```

Comment:
whether we need to report the empty data ?

Reply:
it will report only for the created projects